src/Accounts/Accounts/Account/ConnectAzureRmAccount.cs

```diff
@@ -750,6 +750,7 @@ public void OnImport()
                 AzureSession.Instance.RegisterComponent(nameof(AzureCredentialFactory), () => new AzureCredentialFactory());
                 AzureSession.Instance.RegisterComponent(nameof(MsalAccessTokenAcquirerFactory), () => new MsalAccessTokenAcquirerFactory());
                 AzureSession.Instance.RegisterComponent<ISshCredentialFactory>(nameof(ISshCredentialFactory), () => new SshCredentialFactory());
+                AzureSession.Instance.RegisterComponent<ITestCoverage>(nameof(ITestCoverage), () => new TestCoverage());
```

Comment:
To minimize the impact to customers, let's wrap this with `#if DEBUG || TESTCOVERAGE` as well.
Also, if we add it here, we can remove the `#if` in `TestCoverage.cs`